CharlieBackend.Core/Entities/StudentGroup.cs

```diff
@@ -6,7 +6,7 @@ namespace CharlieBackend.Core.Entities
     public partial class StudentGroup : BaseEntity
     {
 
-        public long CourseId { get; set; }
+        public long? CourseId { get; set; }
```

Comment:
Ig it was fixed in dev, u sure about nullable ?

Reply:
Its same fix like collegue did. I have old dev version. Will update it before merge